src/Behat/Testwork/ServiceContainer/Configuration/ConfigurationLoader.php

```diff
@@ -190,6 +190,8 @@ protected function loadEnvironmentConfiguration()
      * @return array
      *
      * @throws ConfigurationLoadingException If config file is not found
+     *
+     * @phpstan-impure
```

Comment:
This needs to be added so that PHPStan knows that this function can modify the $profileFound variable